src/test/java/org/apache/maven/plugins/dependency/AbstractDependencyMojoTestCase.java

```diff
@@ -35,33 +36,52 @@
 import org.eclipse.aether.RepositorySystemSession;
 import org.eclipse.aether.repository.LocalRepository;
 import org.eclipse.aether.repository.LocalRepositoryManager;
+import org.junit.Before;
 import org.sonatype.plexus.build.incremental.DefaultBuildContext;
 
 public abstract class AbstractDependencyMojoTestCase extends AbstractMojoTestCase {
 
     protected File testDir;
-
     protected DependencyArtifactStubFactory stubFactory;
 
-    protected void setUp(String testDirectoryName, boolean createFiles) throws Exception {
-        setUp(testDirectoryName, createFiles, true);
-    }
-
-    protected void setUp(String testDirectoryName, boolean createFiles, boolean flattenedPath) throws Exception {
-        // required for mojo lookups to work
+    @Override
+    @Before
```

Comment:
This is misleading.
This is still JUnit3 TestCase (and its successors are executed as such) and JUnit's annotations have no power here (or there).

Reply:
- https://github.com/apache/maven-javadoc-plugin/pull/346#issuecomment-2521034423